packages/viewer/src/modules/tree/RenderTree.ts

```diff
@@ -31,11 +32,11 @@ export class RenderTree {
 
   public buildRenderTree(
     geometryConverter: GeometryConverter,
-    callback?: () => void
+    countCallback: (count: number) => void
```

Comment:
I would make this optional, so that when no callback is wanted, none is provided instead of an empty one (like in the case of ObjLoader)